conans/client/tools/win.py

```diff
@@ -463,10 +463,11 @@ def vcvars_command(conanfile=None, arch=None, compiler_version=None, force=False
     return " ".join(command)
 
 
-def vcvars_dict(settings, arch=None, compiler_version=None, force=False, filter_known_paths=False,
-                vcvars_ver=None, winsdk_version=None, only_diff=True, output=None):
+def vcvars_dict(conanfile=None, arch=None, compiler_version=None, force=False,
```

Comment:
This will break positional argument: https://docs.conan.io/en/latest/reference/tools.html#tools-vcvars-dict

Reply:
What breaks is forwarding it to `vcvars_command` as a named argument, after the last commit it should be fine.